src/entry-point-client/PageList.tsx

```diff
@@ -49,16 +49,16 @@ const PageList: PageInfo[] = [
     contents: <MultiFacilityPage />,
   },
   {
-    path: "/inspect-model",
-    title: getPageTitle("Inspect epidemic model output"),
+    path: "/multi-facility/facility",
```

Comment:
Is there a way to set a dynamic parameter here (i.e. the facility's id) such that we can pass it as a prop to the `FacilityPage` component.  When present, it would load that facility's data.  When not present, it would load the blank form.  Not sure if that's the way things are handled in the React world or not.

Reply:
I've never done that in React myself, instead relying on props to determine state. You could check if react-router has anything to that effect but I don't think, from my still non-expert perspective, that that's canonical.

Reply:
Cool, cool.  Won't sweat it for now and will figure out a way to pass along the info I need